tests/plugins/tableresize/manual/iebordercollapse.md

```diff
@@ -0,0 +1,26 @@
+@bender-ui: collapsed
+@bender-tags: bug, 2823, 4.14.0
+@bender-ckeditor-plugins: wysiwygarea, toolbar, table, tableresize, basicstyles, undo
+
+## Procedure
+1. Move mouse over the right table border (border after the last column).
+
+	### Expected
+
+	* Move cursor is visible.
+
+		Note: cursor can become visible slightly _after_ the right border.
+
+	### Unexpected
```

Comment:
Something wrong with formatting here:

![](https://user-images.githubusercontent.com/1061942/76955040-be857800-6911-11ea-9c5e-1f0a47b54dac.png)

Reply:
Same for the next **Unexpected** below too.